feeds/oregon-gtfs.com.dmfr.json

```diff
@@ -1413,6 +1266,84 @@
           ]
         }
       ]
+    },
+    {
+      "spec": "gtfs",
+      "id": "f-point~or-us",
```

Comment:
We have been adding a new way to handle situations where one feed takes the place of multiple previous ones. You can add a `supersedes_ids` property with an array of the previous feed Onestop IDs. For example:
 https://github.com/transitland/transitland-atlas/blob/ca9e9eb40090b5da6a2b195f1990c5909027d6c4/feeds/avta.com.dmfr.json#L7